### PR TITLE
fix(tests): Skip onprem unsupported policy types

### PR DESCRIPTION
**Pull request by jacalvo**
* Do not run cloud policy types tests with `tf_acc_onprem_secure` tag

* Use TypeSet instead of TypeList for rules

### CODE REVIEW

File: sysdig/resource_sysdig_secure_custom_policy_test.go [hunk 1]
```diff
@@ -10,59 +10,62 @@ import (
...
+		},
+		{
+			Config: customPoliciesWithKillAction(rText()),
+		},
+		{
+			Config: customPoliciesWithDisabledRules(rText()),
+		},
+	}
+
+	if !buildinfo.OnpremSecure {
```
Comment (rosenbloomb-sysdig) [inline sysdig/resource_sysdig_secure_custom_policy_test.go:53]: Love this method. I was thinking we would have to separate out these steps into new file. Great job

**Review by rosenbloomb-sysdig — APPROVED** [review]
Awesome!

**Review by papajulio — APPROVED** [review]

**Comment by dbonf** [comment]
> Change rules order in customPolicyWithName() to fix TestAccCustomPolicy

don't you think this can happen in a real life scenario? if order is not important maybe you want to use TypeSet instead of TypeList [here](https://github.com/sysdiglabs/terraform-provider-sysdig/blob/f6815754e4e0d44fe4c35397a83964363f5ff258/sysdig/resource_sysdig_secure_custom_policy.go#L54)

**Comment by jacalvo** [comment]
> > Change rules order in customPolicyWithName() to fix TestAccCustomPolicy
> 
> don't you think this can happen in a real life scenario? if order is not important maybe you want to use TypeSet instead of TypeList [here](https://github.com/sysdiglabs/terraform-provider-sysdig/blob/f6815754e4e0d44fe4c35397a83964363f5ff258/sysdig/resource_sysdig_secure_custom_policy.go#L54)

@dbonf I've pushed 2d8dbed changing to `TypeSet` and reverting the no longer necessary rule order change in the test. Please let me know if it's ok to edit the description and go ahead with the merge or better to split in two different PRs as this change may be a bit out of scope of the original purpose.

**Review by rosenbloomb-sysdig — APPROVED** [review]
Great job!

**Review by jcelaya — APPROVED** [review]
:heart:

**Review by storres1912 — APPROVED** [review]
Thank you @jacalvo!

**Review by kmvachhani — APPROVED** [review]

**Review by miguelgordo — APPROVED** [review]

**Pull request merged**